### DISCUSSION TITLE
Accessing journals with Android

### DISCUSSION THREAD

**Discussion by lamyergeier** [jrnl-org/jrnl#1470] [General]
Relevant Past Issues:

[[Mobile Support? · Issue #652 · jrnl-org/jrnl](https://github.com/jrnl-org/jrnl/issues/652)](https://github.com/jrnl-org/jrnl/issues/652)
[[jrnl Android app · Issue #329 · jrnl-org/jrnl](https://github.com/jrnl-org/jrnl/issues/329)](https://github.com/jrnl-org/jrnl/issues/329)

I had to open a new issue as the old issues got locked. Does someone know of Android app that can be compatible with jrnl file that I can sync using syncthing?

**Comment by micahellison** [comment]
Hi there @lamyergeier, it depends on your use case. I use Drive to sync a journal and can read it just fine on my Android phone, but I don't enter data back into it. With the directory [journal type](https://jrnl.sh/en/stable/journal-types/), it can be a little more manageable to read from the phone since there's a text file for each day. And if you do use a text editor on your phone, your changes should sync back just fine.

**Reply by qazip** [reply]
What if the journal is encrypted? Is there a way to encrypt/decrypt on android?

**Reply by micahellison** [reply]
I'm looking into this more, and I think this might be possible by installing jrnl on your android device. Perhaps by installing Termux then Python then pipx, but I've never tried it. I'd love to hear back from anybody who has.

**Comment by xeruf** [comment]
Maybe one could enhance an existing mobile text editor to highlight and insert jrnl timestamps.
I am thinking of Orgzly which is meant for org-mode but could be very useful with a few modifications, as org-mode has the same timestamp format (on desktop I edit jrnl files in Emacs Org Mode as well).

**Comment by eloyesp** [comment]
Hi, [Diary](https://github.com/billthefarmer/diary) is an existing android App that, as far as I read, should be mostly compatible, the only difference being that the file extension used for the text files (on the folder jrnl). They store data as "markdown" with the `md` extension.

I'm looking for a solution on the opposite direction, I'm using that application on mobile, but would like something like jrnl on the computer.

So adding a config option to use markdown format for the folder might make both applications a great combo.
